src/components/organisms/Header/BackButtonHeader/index.tsx

```diff
@@ -29,13 +28,15 @@ function BackButtonHeader({ pageName, isPending, onClick }: HeaderProp) {
           <PlusSvg color="#70767E" />
         </button>
       )}
-      {pathName === '/mypage' ? (
+      {pathName === '/mypage' && (
         <Link href={'/notification'}>
           {isPending ? <OnNotificationSvg /> : <NotificationSvg />}
         </Link>
-      ) : (
-        pathName.startsWith('/records') ?? <div className={styles.blank} />
       )}
+      {(pathName.startsWith('/follow') ||
+        pathName.startsWith('/setting') ||
+        pathName.startsWith('/map') ||
+        pathName.startsWith('/post')) && <div className={styles.blank} />}
```

Comment:
```suggestion
const blankPaths = ['/follow', '/setting', '/map', '/post'];
const isBlankPaths = pathsToCheck.some(path => pathName.startsWith(path));

isBlankPaths && <div className={styles.blank} />}

```
이런 식으로도 사용할 수 있을 것 같습니다!

Reply:
와 이런 방법도 있군요
너무 좋습니다 👍🏻